expo-flutter/ios/Podfile

```diff
@@ -39,6 +39,16 @@ end
 
 target 'ExpoFlutter' do
   use_expo_modules!
+# @generated begin expo-flutter-view - expo prebuild (DO NOT MODIFY) sync-a6fad925eb19db4b427cb412b9bf06f47580494d
+  pod 'Flutter', :podspec => '../modules/expo-flutter-view/ios/Flutter/Flutter.podspec'
+  pod 'FlutterPluginRegistrant', path: '/Users/mazhnik/Codes/p-mazhnik/flutter-embedding/expo-flutter/modules/expo-flutter-view/ios/Flutter/plugins/FlutterPluginRegistrant'
```

Comment:
full path leaked

Reply:
Yeah I fixed it in the latest commits, squash and merge will prevent this from going to the main branch. 

> I also noticed that you're removing the --clean flag

I just don't think the `clean` flag is needed for each change. It takes a long time for me to reinstall all pod dependencies when using `clean`, and it is unnecessary to use it every time (although not prohibited)

> xcodebuild: error: The workspace named "expoflutter" does not contain a scheme named "ExpoFlutter". 

hm, I am looking at the PR changes and don't see anything related. I can quickly setup build in GH Actions to check

Reply:
Working with scheme written in lower case: https://github.com/p-mazhnik/flutter-embedding/actions/runs/11011828146/job/30576763675
`--scheme expoflutter`

Reply:
I think I lost it when I ran prebuild with --clean